### ISSUE TITLE
recoll doesn't work

### ISSUE THREAD

**Issue by becker44a** [linuxmint/mint22-beta#128]
I installed the full-text search tool "recoll" from the default repos.  It installed "Recoll 1.36.1 + Xapian 1.4.22".  When launched, it offers to index immediately, which I allowed.  To this point, it behaves normally.  After the indexing, I entered a search term, and clicked the Search button.  At this point, recoll caused a forcible logout.  It doesn't matter what search term is entered, or whether it should be found or not - it always does an immediate logout.

Testing in a VirtualBox 6.1.48 Virtual Machine.  Running under Mint 20.3 on an AMD Ryzen 5800H CPU.

To see if this was an upstream issue, I downloaded Ubuntu 24.04, and installed it in another VM (what an unpleasant experience!).  Using apt-get, I installed Recoll (same version as above).  In Ubuntu, Recoll does NOT cause a logout when doing a search.

As always, I will be happy to perform any additional tests that would be helpful.

**Comment by becker44a** [comment]
Just found Issue #35 - after reading that one, my recoll issue may be another instance of that bug.

**Comment by LinuxOnTheDesktop** [comment]
[Here is](https://framagit.org/medoc92/recoll/-/issues) the Recoll issue tracker, in case we need it.

**Comment by mtwebster** [comment]
Please provide more info - what Mint 22 edition (mate, xfce or cinnamon), more system details, etc..

**Comment by becker44a** [comment]
Mint 22 Cinnamon edition, VM settings: 4GB RAM, 4 Virtual CPUS,  Vdisk: 40G statically allocated

Installation dialog:
.) Initiate install of Linux Mint
  Responses:
  Language: English
  Keyboard: English(US)
  Install Multimedia Codecs:  enable
  Installation type:  something else
  Partitioning:
    sda1: 1M      biosgrub
    sda2: 1536M   EXT4, /boot
    sda3: 20G     EXT4, /
    sda4: rmndr   EXT4, /home (appx 16G)
  Note: complained about missing EFI ptn: 
    May create issue installing boot loader.  wait & see. . .  
    Installed and booted no problem.
  Location:  New York
  Name:xxxxxxxx
  Computer: lx22
  User:  alan
  Password:  xxxxxxxxx
  Encrypt home folder: no
  Login:  automatic
  Proceed w/ install, reboot when finished.

I saw some indications elsewhere that this issue may be related to running in a VM.  I have an old HP notebook for experiments, so trying a similar load to see if recoll fails there or not.  Will update when some results.

**Comment by becker44a** [comment]
Completed load of Mint 22 Cinnamon on the HP notebook
  HP Pavilion dv6-6117dx 15.6" HD AMD A8-3500M 4GB 500GB
  Specs:  CPU A8-3500M APU 1.5 GHz   Ram 4 Gb   Hdd 500 Gb
NOTE:  Mint 22 Cinnamon Install .iso would not boot unless added "nomodeset" kernel parameter

Completed partitioning, User/Account setup, loading & reboot.  Set Software Sources.
Before doing ANY other settings, installs, adjustments, did the following:
.) Recoll test:
  . Install recoll
  . Launch recoll
  . Initiate indexing on alan home
  . Search for "alan"
  . Successfully found 2 instances of alan
  . DID NOT terminate user session ! ! !

This seems to confirm that the issue appears when running in a VM, and not in bare-metal installs.

**Comment by becker44a** [comment]
In my VM  loaded w/ Mint 22 Cinnamon, I tested opening Pix and Rythmbox, and a search in recoll.  All crashed the user session, returning to the login screen.
After a reboot, I removed gstreamer1.0-vaapi, which also removed mint-meta-codecs.
Now Pix and Rythmbox open normally, and Recoll will search as it should.  Some limited further testing did not reveal any failures.

As a workaround, this appears to correct the issue.  I don't know what is broken or missing as a consequence.
